PowerShell/JumpCloud Commands Gallery/Mac Commands/Application Installs/Mac - Generic DMG Installer.md

```diff
@@ -19,164 +20,118 @@ mac
 
 # *NOTE* this template is only designed to work with DMG files and does not support .pkg or .zip files.
 
-DownloadUrl=""
-
-### Modify below this line at your own risk!
```

Comment:
I think we should keep this line just to break up the script from what we want them to change